client/my-sites/plans/jetpack-plans/use-item-price.ts

```diff
@@ -191,6 +192,15 @@ const useItemPrice = (
 		originalPrice = item.displayPrice || -1;
 	}
 
+	if (
```

Comment:
Currently hardcoding the price (instead of fetching from API) until we are ready with the product in Store DB